pkg/remote/builtin/github/helper.go

```diff
@@ -58,7 +58,7 @@ func GetUserEmail(client *github.Client) (*github.User, error) {
 	// WARNING, HACK
 	// for out-of-date github enterprise editions the primary
 	// and verified fields won't exist.
-	if !strings.HasPrefix(*user.HTMLURL, DefaultURL) && len(emails) != 0 {
+	if !strings.HasPrefix(*user.HTMLURL, defaultURL) && len(emails) != 0 {
```

Comment:
I think we can leave this as a constant (`DefaultURL`) because basically we are saying that if github enterprise, and no email, allow the user to proceed.